docs/docs/chrome-extension/features.md

```
@@ -4,34 +4,34 @@
 This chat session is **private**, and won't be visible to other users.
 
 All open-source repositories are supported.
-For private repositories, you will also need to install PR-Insight Pro, After installation, make sure to open at least one new PR to fully register your organization. Once done, you can chat with both new and existing PRs across all installed repositories.
+For private repositories, you will also need to install Khulnasoft Merge. After installation, make sure to open at least one new PR to fully register your organization. Once done, you can chat with both new and existing PRs across all installed repositories.
 
 #### Context-aware PR chat
 
-PR-Insight constructs a comprehensive context for each pull request, incorporating the PR description, commit messages, and code changes with extended dynamic context. This contextual information, along with additional PR-related data, forms the foundation for an AI-powered chat session. The insight then leverages this rich context to provide intelligent, tailored responses to user inquiries about the pull request.
+Khulnasoft Merge constructs a comprehensive context for each pull request, incorporating the PR description, commit messages, and code changes with extended dynamic context. This contextual information, along with additional PR-related data, forms the foundation for an AI-powered chat session. The agent then leverages this rich context to provide intelligent, tailored responses to user inquiries about the pull request.
 
 <img src="https://khulnasoft.com/images/pr_insight/pr_chat_1.png" width="768">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)